eng/pipelines/official/jobs/prepare-signed-artifacts.yml

```diff
@@ -62,9 +62,9 @@ jobs:
     continueOnError: true
     condition: succeededOrFailed()
 
-  - task: PublishBuildArtifacts@1
-    displayName: Publish Artifact BuildLogs
+  - task: PublishPipelineArtifact@1
```

Comment:
Pipeline artifacts can only be uploaded once and from a single job. The prepare-signed-artifacts.yml file is used as a template but has a fixed `ArtifactName`. Today it appears to be only used once hence I don't think we'll have a conflcit but given it's a template we should parameterize the name so there aren't conflicts.